tools/accuracy_checker/accuracy_checker/adapters/pose_estimation_3d.py

```diff
@@ -0,0 +1,254 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import OrderedDict
+
+import cv2
+import numpy as np
+
+from ..adapters import Adapter
+from ..adapters.pose_estimation import HumanPoseAdapter
+from ..config import ConfigValidator, StringField
+from ..representation import PoseEstimationPrediction, PoseEstimation3dPrediction
+
+AVG_PERSON_HEIGHT = 180
+
+# pelvis (body center) is missing, id == 2
+map_id_to_panoptic = [1, 0, 9, 10, 11, 3, 4, 5, 12, 13, 14, 6, 7, 8, 15, 16, 17, 18]
+
+limbs = [[18, 17, 1],
+         [16, 15, 1],
+         [5, 4, 3],
+         [8, 7, 6],
+         [11, 10, 9],
+         [14, 13, 12]]
+
+
+class HumanPose3dAdapter(Adapter):
+    __provider__ = 'human_pose_estimation_3d'
+    prediction_types = (PoseEstimation3dPrediction,)
+
+    def __init__(self, launcher_config, label_map=None, output_blob=None):
+        super().__init__(launcher_config, label_map, output_blob)
+        launcher_2d_config = OrderedDict([
+            ('type', HumanPoseAdapter.__provider__),
```

Comment:
@eaidova Added manual config creation to supress warning about unknown `features_3d_out` value in config.

Reply:
ok, or you can just replace in `HumanPoseAdapter`
```
        super().validate_config(on_extra_argument=ConfigValidator.WARN_ON_EXTRA_ARGUMENT)
```
on
```
 super().validate_config(on_extra_argument=ConfigValidator.IGNORE_ON_EXTRA_ARGUMENT)
```
or remove `validate_config` at all, as I remember, ignoring is default case for adapters.

Reply:
Checked adapters, there are no common rule: some of them warn, some ignores, some emit error. Think is ok to show warning on unknown arguments, so left as is.

Reply:
ok, it is right